### ISSUE TITLE
Taking only physical spell crit into concideration

### ISSUE THREAD

**Issue by WrIgHtaufGanovenjagd** [BreakBB/ExtendedCharacterStats#180]
https://github.com/BreakBB/ExtendedCharacterStats/blob/7ee6b55341c7f4df53f37bd0b0538e91c4aab2fa/Modules/Data/SpellCrit.lua#L20

that line uses "GetSpellCritChance(1)" which gets the spell crit chance for physical spells.
those usually aren't affected by items that give additional crit chance what leads to Spellcrit for specific schools displaying the wrong value.

if i'm not wrong that line should instead be "crit = crit + GetSpellCritChance(school) + itemBonus + setBonus".

**Comment by BreakBB** [comment]
Yes @WrIgHtaufGanovenjagd you are right. We also discussed this in #178 and I will get out a fix for this in a bit.
